dsync/remote.go

```diff
@@ -29,6 +29,14 @@ const (
 )
 
 // Remote is an interface for a source that can be synced to & from
+// The main confusing thing about this package is the difference between
```

Comment:
We shouldn't document confusing things. Let's file issues & make fixes instead. I'd nix this comment entirely in favour of an issue